controllers/suite_test.go

```diff
@@ -60,6 +60,9 @@ var _ = BeforeSuite(func(done Done) {
 	err = controlplanev1beta1.AddToScheme(scheme.Scheme)
 	Expect(err).NotTo(HaveOccurred())
 
+err = controlplanev1beta1.AddToScheme(scheme.Scheme)
```

Comment:
Both these lines look to be a duplicate of the lines above so I think you can remove it.

Reply:
weird, must be a bug in the sdk code generation. I removed them.